packages/strapi-plugin-graphql/hooks/graphql/index.js

```diff
@@ -54,7 +54,7 @@ module.exports = strapi => {
         // Load APIs configurations.
         new Promise((resolve, reject) => {
           glob(
-            './api/*/config/*.graphql',
+            './api/*/config/*{.graphql,.graphql.js}',
```

Comment:
🤔 Why not add just the optional extension?; it the real objective isn't?. Not tested, but I think it should be something like:

```
'./api/*/config/*.graphql?(.js)'
```

Reply:
Yeah that might make more sense. I'm not so good with globs 😄